### PR TITLE
Promote Sysctl to ga

### PR DESCRIPTION
**Pull request by pacoxu**
Wait for e2e promotion to conformance testing

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/sysctl-cluster.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ content_type: task
 ---
 
 <!-- overview -->
-{{< feature-state for_k8s_version="v1.12" state="beta" >}}
+{{< feature-state for_k8s_version="v1.21" state="GA" >}}
```
Comment (neolit123) [inline content/en/docs/tasks/administer-cluster/sysctl-cluster.md:9]: i don't think this "feature-state" makes a lot of sense here.
`sysctl` is not a tool originating from the Kubernetes project and its usage cannot fall under a GA or Beta state.

Comment (neolit123) [reply content/en/docs/tasks/administer-cluster/sysctl-cluster.md:9]: it was added in this PR:
https://github.com/kubernetes/website/pull/8804

**Comment by tengqm** [comment]
/hold
Feel free to unhold when E2E conformance test is ready.

File: content/en/docs/tasks/administer-cluster/sysctl-cluster.md (hunk 1 above)
Comment (pacoxu) [reply content/en/docs/tasks/administer-cluster/sysctl-cluster.md:9]: So, I should remove it?

I tried to summarize the history of sysctls in https://github.com/kubernetes/enhancements/issues/34#issuecomment-782553906

What we have done in 1.21 is lock the feature gate to True and promote e2e test cases(which is in process).

Comment (neolit123) [reply content/en/docs/tasks/administer-cluster/sysctl-cluster.md]: i don't maintain this page, so i was just commenting.
sig-docs may have feedback on this.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -331,6 +330,8 @@ different Kubernetes components.
 | `SupportPodPidsLimit` | `false` | Alpha | 1.10 | 1.13 |
 | `SupportPodPidsLimit` | `true` | Beta | 1.14 | 1.19 |
 | `SupportPodPidsLimit` | `true` | GA | 1.20 | - |
+| `Sysctls` | `true` | Beta | 1.11 | |
```
Comment (ehashman) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:333]:
```suggestion
| `Sysctls` | `true` | Beta | 1.11 | 1.20 |
```

**Comment by reylejano** [comment]
Hi @pacoxu, looks like this PR is waiting for the e2e promotion to conformance testing in kubernetes/kubernetes#99734 to remove the hold.
Just wanted to share upcoming doc related dates for the 1.21 release:
- <strike> Docs Ready for Review deadline is March 24</strike> done
- Docs Ready to Merge deadline is March 31 (waiting for kubernetes/kubernetes#99734)

**Comment by pacoxu** [comment]
@reylejano  will update this pr after that e2e promotion PR merged

**Comment by reylejano** [comment]
@pacoxu Can you squash your commits and rebase
Thank you

**Comment by pacoxu** [comment]
Squanshed and rebased.

**Review by ehashman — COMMENTED** [review]
/lgtm

**Comment by pacoxu** [comment]
/hold cancel
the test case promotion was merged

File: content/en/docs/tasks/administer-cluster/sysctl-cluster.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tasks/administer-cluster/sysctl-cluster.md]: The feature gates page lists `Sysctls` as a feature. Is that incorrect?

**Pull request merged**